client/lib/pages/Camera.dart

```diff
@@ -553,7 +557,7 @@ class _CameraState extends State<Camera>
         ),
         Padding(
           padding: const EdgeInsets.only(left: 8.0),
-          child: Text("Confidence : "+_recognitions.map((res){ return res["confidence"];}).toList()[0].toString()),
+          child: Text("Confidence : "+_recognitions.map((res){ return res["confidence"];}).toList()[0].toStringAsFixed(2).toString()),
```

Comment:
If you just want the confidence of the first recognition this can be written much simpler:
`_recognitions.first["confidence"].toStringAsFixed(2)`